DepositeCalcTests/Tests/LoginPageTests.cs

```diff
@@ -0,0 +1,60 @@
+using DepositeCalcTests.Pages;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using System;
+using System.Threading;
+
+namespace DepositeCalcTests.Tests
+{
+    public class LoginPageTests
+    {
+        private IWebDriver driver;
+
+        [SetUp]
+        public void Setup()
+        {
+            ChromeOptions options = new ChromeOptions { AcceptInsecureCertificates = true };
+            driver = new ChromeDriver(options);
+            driver.Manage().Timeouts().PageLoad = TimeSpan.FromSeconds(30);
+            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(5);
+            driver.Url = "https://localhost:5001/";
```

Comment:
фівадфіва жо жаофівжао 
```suggestion
            driver.Manage().Timeouts().PageLoad = TimeSpan.FromSeconds(40);
            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(65);
            driver.Url = "https://localhost:5001/";
```